tests/conftest.py

```diff
@@ -721,17 +715,17 @@ def minicircle_grainstats(
 FULL_IMAGE = RNG.random((GRAINS.shape[0], GRAINS.shape[1]))
 
 
-@pytest.fixture
+@pytest.fixture()
 def test_dnatracing() -> dnaTrace:
     """Instantiate a dnaTrace object."""
     return dnaTrace(image=FULL_IMAGE, grain=GRAINS, filename="Test", pixel_to_nm_scaling=1.0)
 
 
-@pytest.fixture
+@pytest.fixture()
 def minicircle_dnatracing(
     minicircle_grain_gaussian_filter: Filters, minicircle_grain_coloured: Grains, dnatracing_config: dict
 ) -> dnaTrace:
-    """dnaTrace object instantiated with minicircle data."""
+    """DnaTrace object instantiated with minicircle data."""
```

Comment:
Is there a way to over-ride the pydocstyle linter rules here to allow the lower case d on dnaTrace as this is the class name pulled from the code?

Reply:
Yes there is, if we can identify the rule then at the end of the line we add `# noqa: <rule number>`. I'll look that up now. :+1:

Reply:
Done it was [D403 (first-line-capitalized)](https://docs.astral.sh/ruff/rules/first-line-capitalized/), thanks :+1: